apps/dokploy/templates/triggerdotdev/docker-compose.yml

```
@@ -0,0 +1,107 @@
+x-webapp-env: &webapp-env
+  LOGIN_ORIGIN: &trigger-url ${TRIGGER_PROTOCOL:-http}://${TRIGGER_DOMAIN:-localhost:3040}
+  APP_ORIGIN: *trigger-url
+  DEV_OTEL_EXPORTER_OTLP_ENDPOINT: &trigger-otel ${TRIGGER_PROTOCOL:-http}://${TRIGGER_DOMAIN:-localhost:3040}/otel
+  ELECTRIC_ORIGIN: http://electric:3000
+
+x-worker-env: &worker-env
+  PLATFORM_HOST: webapp
+  PLATFORM_WS_PORT: 3030
+  SECURE_CONNECTION: "false"
+  OTEL_EXPORTER_OTLP_ENDPOINT: *trigger-otel
+
+volumes:
+  postgres-data:
+  redis-data:
+
+networks:
+  webapp:
+
+services:
+  webapp:
+    image: ghcr.io/triggerdotdev/trigger.dev:${TRIGGER_IMAGE_TAG:-v3}
+    restart: ${RESTART_POLICY:-unless-stopped}
+    env_file:
+      - .env
+    environment:
+      <<: *webapp-env
+    ports:
+      - ${WEBAPP_PUBLISH_IP:-127.0.0.1}:3040:3030
+    depends_on:
```

Trigger.dev use the port 3000 to listen the webapp
```suggestion
    ports:
      - 3000
    depends_on:
```